src/info-notification.vala

```diff
@@ -21,102 +21,102 @@ using Notify;
 
 public class IndicatorSound.InfoNotification: Notification
 {
-	protected override Notify.Notification create_notification () {
-		return new Notify.Notification (_("Volume"), "", "audio-volume-muted");
-	}
+    protected override Notify.Notification create_notification () {
+        return new Notify.Notification (_("Volume"), "", "audio-volume-muted");
+    }
 
-	public void show (VolumeControl.ActiveOutput active_output,
-	                  double volume,
-	                  bool is_high_volume) {
-		if (!notify_server_supports ("x-canonical-private-synchronous"))
-			return;
+    public void show (VolumeControl.ActiveOutput active_output,
+                      double volume,
+                      bool is_high_volume) {
+        if (!notify_server_supports ("x-ayatana-private-synchronous"))
```

Comment:
x-canonical-private-synchronous is used across many other projects. We should not change it.